blocks/filter.py

```diff
@@ -44,7 +44,8 @@ class VariableFilter(object):
     ----------
     roles : list of :class:`.VariableRole` instances, optional
         Matches any variable which has one of the roles given.
-    bricks : list of :class:`.Brick` classes or instances, optional
+    bricks : list of :class:`.Brick` classes or instances,
+        or instance of :class:`.Brick` classes or instances, optional
```

Comment:
Line breaks in type descriptions don't work I think. Just stick with something like
```
bricks : list
    List of ... bla bla
    and more bla bla
```